docs/components/heading-hierarchy.md

```diff
@@ -1,64 +1,183 @@
 ---
-title: Heading Hierarchy
+title: Headings
 permalink: /components/heading-hierarchy
 layout: component
 collection_name: components
 section: components
 status: Proposed
-intro: |-
-  More information can be found at:
+intro: >-
+  Hierarchy refers to the difference in type size and weight between text
+  elements. A successful hierarchy establishes the order of importance of
+  elements on a page, allowing readers to easily scan content and find what
+  they’re looking for.
+
+
+  This page is under active development. Information is being moved into it from
+  these pages:
+
+
+  * http://cfpb.github.io/design-manual/brand-guidelines/typography.html
 
-  * "http://cfpb.github.io/design-manual/brand-guidelines/typography.html
   * https://cfpb.github.io/capital-framework/components/cf-core/#type-hierarchy
 variations:
-  - variation_code_snippet: <marquee>Some sample code will eventually show up here.</marquee>
-    variation_description: The default state is the most default state of them all.
-    variation_name: Default state
-  - variation_code_snippet: <marquee>Some sample code will eventually show up here.</marquee>
-    variation_description: The secondary state is very secondary.
-    variation_name: Secondary state
+  - variation_code_snippet: <h1 class="superheading">Example display heading</h1>
+    variation_description: '<601px wide: Drops to Heading 1'
+    variation_jinja_code_snippet: ''
+    variation_name: Display heading
+    variation_specs: "Element | Variable name | Variable | web value | Print value\n--- | --- | --- | --- | ---\nTypeface | | | Avenir Next Regular | Avenir Next Regular\nWebfont | @webfont-regular: | Arial; | Arial\nText color | @text: | @black; | Black (#101820)\t| 0, 0, 0, 100\nText size | @size-xl: | 48px; | 48px | 38pt\nSmall screen text size | | | 34px | \nLine height | @base-line-height: | unit( @base-line-height-px / @base-font-size-px ); | \nLine height (pixels) | ? |  | 60px | 66pt\nSmall screen line height | | 42px | \nBottom margin | | | | 30px\nExtra-small breakpoint | @bp-xs-max: | 600px; |"
```

Comment:
Nitpick: colons aren't a part of variable names, so shouldn't be included long-term. For example, in this area I'd expect to see `@webfont-regular` instead of `@webfont-regular:`.

Reply:
Thanks @marteki !